site/documentation/content/deployment/create-kubernetes-cluster.md

```diff
@@ -10,19 +10,31 @@ Hono can be deployed to any Kubernetes cluster running version 1.11 or newer. Th
 
 The [Kubernetes web site](https://kubernetes.io/docs/setup/) provides instructions for setting up a (local) cluster on bare metal and/or virtual infrastructure from scratch or for provisioning a managed Kubernetes cluster from one of the popular cloud providers.
 
-## Local Development
+<a name="Local Development"></a>
+## Setting up a local Development Environment
 
 The easiest option is to set up a single-node cluster running on a local VM using the [Minikube](https://github.com/kubernetes/minikube) project.
 This kind of setup is sufficient for evaluation and development purposes.
 Please refer to Kubernetes' [Minikube setup guide](https://kubernetes.io/docs/setup/minikube/) for instructions on how to set up a cluster locally.
 
-When using Minikube, the `minikube tunnel` command should be run in order to support Hono's services being deployed using the *LoadBalancer* type. Please refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#loadbalancer-emulation-minikube-tunnel) for details.
+The recommended settings for a Minikube VM used for running Hono's example setup are as follows:
 
-## Production Environment
+* **cpus**: 2
+* **memory**: 8192 (MB)
+* **kubernetes-version**: v1.13.6
+
+The command to start the VM will look something like this:
+```sh
+minikube start --cpus 2 --memory 8192 --kubernetes-version v1.13.6 --vm-driver kvm2 -p hono
+```
+
+You may want to use another VM driver, depending on your OS. After the Minikube VM has started successfully, the `minikube tunnel` command should be run in order to support Hono's services being deployed using the *LoadBalancer* type. Please refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#loadbalancer-emulation-minikube-tunnel) for details.
```

Comment:
The anchor in the linked minikube docs page seems to have changed. It is now:
https://github.com/kubernetes/minikube/blob/master/docs/networking.md#access-to-loadbalancer-services-using-minikube-tunnel